audformat/core/attachment.py

```diff
@@ -59,6 +60,72 @@ def __init__(
         self.path = path
         r"""Attachment path"""
 
+    @property
+    def files(
+            self,
+    ) -> typing.List[str]:
+        r"""List all files part of the attachment.
+
+        List recursively the relative path
+        of all files that exist
+        under :attr:`audformat.Attachment.path`
+        on hard disc.
```

Comment:
In other places we already use the English spelling.

```suggestion
        on hard disk.
```